dev-python/pyparsing/pyparsing-2.4.5.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 2004-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} pypy pypy3 )
```

Comment:
One empty line below EAPI please.

python3.5 has been dropped from the tree, could you remove support for it? There were some changes to pypy as well, but I don't remember which. If nothing comes to your mind, just leave it.